src/shared/utilities/tsUtils.ts

```diff
@@ -4,7 +4,13 @@
  */
 
 export const getPropAs = <T>(obj: any, key: string) => {
-    return ((obj as any) as {
-        [key: string]: T
-    })[key]
+    return (
+        obj as any as {
+            [key: string]: T
+        }
+    )[key]
 }
+
+type NoSymbols<T> = { [Property in keyof T]: Property extends symbol ? never : Property }[keyof T]
+export type InterfaceNoSymbol<T> = Pick<T, NoSymbols<T>>
```

Comment:
This seems unused?

Reply:
Used in the S3 client. The implementations had public symbol fields that were used exclusively for debugging. The alternative is just to mark the fields private, though it doesn't matter much either way.